ui-tests/src/test/java/org/openmrs/reference/UsingBackButtonInMergePatientTest.java

```diff
@@ -1,23 +1,23 @@
 package org.openmrs.reference;
 
 import org.junit.Before;
+import org.junit.After;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.helper.TestPatient;
 import org.openmrs.reference.page.*;
```

Comment:
explicitly import classes since this would avail all pages which is not right

Reply:
@kdaud I have not understood your comment.

Reply:
In view of the code pointed to.

Reply:
> I have not understood your comment.

Check [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/610b3dbcba58f081d13ea0e842994f852477ff8c/ui-tests/src/test/java/org/openmrs/reference/UsingBackButtonInMergePatientTest.java#L9)

Reply:
@kdaud worked on it already